### PR TITLE
updated html for show page

### PR DESCRIPTION
**Pull request by angelacoates**
Hey Gavin,

I completed updating the html for the show page.

### CODE REVIEW

File: app/assets/stylesheets/global.scss [hunk 1]
```diff
@@ -12,3 +13,21 @@
...
   color: red;
 }
+
+
+.page-header{
+
+  color:rgb(196, 145, 42);
+  font-family: sans-serif;
+  font-variant: small-caps;
+
```
Comment (gstark) [inline app/assets/stylesheets/global.scss]: Please tidy the blank lines.

File: app/views/items/show.html.erb [hunk 2]
```diff
@@ -1,35 +1,46 @@
...
 
-  <dt>Created by:</dt>
-  <dd><%= @item.created_by %></dd>
 
-  <dt>Category:</dt>
-  <dd><%= @item.category %></dd>
+<div class="media">
+  <div class="media-left">
+    <a href="#">
+  <%= image_tag(@item.image_url(:medium)) %>
```
Comment (gstark) [inline app/views/items/show.html.erb]: Indentation.

**Comment by gstark** [comment]
👍 :shipit: after minor issues

**Comment by angelacoates** [comment]
Hey Gavin,

I fixed the spacing and indentation, also completed styling for show page.

**Comment by gstark** [comment]
👍 :shipit: !

**Pull request merged**